### PR TITLE
JIRA78899 Arch Diag and Install parity

### PR DESCRIPTION
**Pull request by mmcelroy91**
https://jira.d2iq.com/browse/D2IQ-78899

Docs Parity - Creating a new Architecture Diagram based on new platform choices, addons, etc.  Also, bringing Install page forward to 2.2 Docs.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4379.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Comment by mmcelroy91** [comment]
Arch diagram was corrupt so I am closing this to re-create.

**Comment by mmcelroy91** [comment]
Trying to get the non-corrupt architecture drawing to upload and having some trouble, so hold off reviewing this one

File: pages/dkp/konvoy/2.2/architecture/index.md
Comment (gvasquezvargas) [inline pages/dkp/konvoy/2.2/architecture/index.md:13]: So Vale stops complaining and you get a beautifully green PR: 
```suggestion
The topics in this section provide a brief overview of the native Kubernetes architecture, a simplified view of the DKP architecture - both Essential and Enterprise versions. Also, the it shows the operational workflow for a DKP cluster.
```

File: pages/dkp/konvoy/2.2/Install/index.md
Comment (gvasquezvargas) [inline pages/dkp/konvoy/2.2/Install/index.md:6]: Not sure, but I think this should be "Install Konvoy on a cluster" OR "Create a Konvoy cluster".

File: pages/dkp/konvoy/2.2/Install/index.md
Comment (gvasquezvargas) [inline pages/dkp/konvoy/2.2/Install/index.md:27]: I think we should take out the AWS reference as we want to support other providers? Or if this is AWS-specific, surface that in the title?

File: pages/dkp/konvoy/2.2/Install/index.md
Comment (gvasquezvargas) [inline pages/dkp/konvoy/2.2/Install/index.md:49]: Do we have/need a command here? Or is it self-explanatory? Not sure...

File: pages/dkp/konvoy/2.2/architecture/index.md
Comment (gvasquezvargas) [inline pages/dkp/konvoy/2.2/architecture/index.md:55]:
```suggestion
- The `kubelet` agent runs on each node in the cluster to ensure that the containers created by Kubernetes for a pod are running and in a healthy state.
```

**Review by gvasquezvargas — APPROVED** [review]

**Review by gvasquezvargas — APPROVED** [review]

File: pages/dkp/konvoy/2.2/Install/index.md
Comment (mmcelroy91) [reply pages/dkp/konvoy/2.2/Install/index.md:27]: I think these are part of 1.8 instructions and I should probably modify this to say the example is AWS specific.  I know I included the "Choose your infrastructure" link after initial installation to further the instructions, but honestly, you're right, this should be worded different.

File: pages/dkp/konvoy/2.2/Install/index.md
Comment (mmcelroy91) [reply pages/dkp/konvoy/2.2/Install/index.md:49]: I think it is just saying to accept the "defaults" at this point of installation, but I will defer to Dimitri to verify.

File: pages/dkp/konvoy/2.2/Install/index.md
Comment (mmcelroy91) [reply pages/dkp/konvoy/2.2/Install/index.md:1]: I believe there is currently a Kommander "Install" page.  Is that what you mean?  We were just missing the "Install" page for Konvoy... understand they are essentially DKP now.  That's where I probably need more help with verbiage.

File: pages/dkp/konvoy/2.2/architecture/index.md
Comment (mmcelroy91) [reply pages/dkp/konvoy/2.2/architecture/index.md:73]: Correct... wasn't sure if once we resolve JIRA78929 regarding those being called Platform Applications now, should we link to that section instead?

File: pages/dkp/konvoy/2.2/architecture/index.md
Comment (mmcelroy91) [reply pages/dkp/konvoy/2.2/architecture/index.md:17]: fixed, waiting to upload until Jenkins is running properly.

File: pages/dkp/konvoy/2.2/Install/index.md
Comment (cneth) [inline pages/dkp/konvoy/2.2/Install/index.md:62]: This is not needed with 2.x, the state is stored in the cluster.


```suggestion
```

File: pages/dkp/konvoy/2.2/Install/index.md
Comment (cneth) [inline pages/dkp/konvoy/2.2/Install/index.md:70]: There needs to be an additional before this one, which is is to install Kommander, as without it, all you have is a very basic Kubernetes cluster (the very bottom layer of the diagram).

Maybe we need to add an '# Install Kommander' bullet here with a pointer to the Kommander install directions?

File: pages/dkp/konvoy/2.2/Install/index.md
Comment (cneth) [inline pages/dkp/konvoy/2.2/Install/index.md:50]: This is redundant with the "Deploy with all the default settings...." below.

```suggestion
```

**Review by cneth — CHANGES_REQUESTED** [review]

File: pages/dkp/konvoy/2.2/architecture/index.md
Comment (dkoshkin) [inline pages/dkp/konvoy/2.2/architecture/index.md:23]: Please remove all occurrences of "master" and replace it with control plane, the term "master" is not used in upstream Kubernetes docs

File: pages/dkp/konvoy/2.2/architecture/index.md
Comment (dkoshkin) [inline pages/dkp/konvoy/2.2/architecture/index.md:3]: What are we trying to describe with this doc? Upstream Kubernetes docs already have https://kubernetes.io/docs/concepts/overview/components/, don't we want to highlight the architecture of Konvoy/DKP here instead?

File: pages/dkp/konvoy/2.2/Install/index.md
Comment (dkoshkin) [inline pages/dkp/konvoy/2.2/Install/index.md:5]: This page is kind of the bottom of the list and not very clear that should be the entrypoint for users.

image

File: pages/dkp/konvoy/2.2/architecture/index.md
Comment (mmcelroy91) [reply pages/dkp/konvoy/2.2/architecture/index.md:3]: I agree... the diagram just looked kind of lonely on the page.  Should I bring the information from Choose Infrastructure to this page instead?  Or to avoid duplicate data, just delete all the extraneous blabber and just have the diagram located after the Intro to DKP page?

File: pages/dkp/konvoy/2.2/Install/index.md
Comment (dlipovetsky) [inline pages/dkp/konvoy/2.2/Install/index.md:51]: image

Numbering is not contiguous. If it should be,  please remove this line.
```suggestion
```

**Comment by dlipovetsky** [comment]
I have many questions about the purpose and content of the Architecture doc. Would you be open to removing the Architecture doc from this PR, so that we can review and merge the Install doc, and then discuss the Architecture doc separately?

File: pages/dkp/konvoy/2.2/Install/index.md
Comment (mmcelroy91) [reply pages/dkp/konvoy/2.2/Install/index.md]: Thanks for that... forgot to change the menu weight!

File: pages/dkp/konvoy/2.2/Install/index.md
Comment (mmcelroy91) [reply pages/dkp/konvoy/2.2/Install/index.md]: Fixed it!  Good catch!

**Comment by mmcelroy91** [comment]
Daniel, when we move to Confluence, we will be happy to change the Architecture diagram for 2.3

**Pull request merged**
